### ISSUE TITLE
go training problems ??

### ISSUE THREAD

**Issue by lyxlty** [AlexeyAB/darknet#1351]
Hi ,AlexeyAB,I have train successed the darknet and got yolov3-tiny_obj_14000.weights,,but when I go to train  :darknet.exe detector train data/obj.data yolov3-tiny_obj.cfg yolov3-tiny_obj_14000.weights
![image](https://user-images.githubusercontent.com/11659778/43771108-bd8171c8-9a70-11e8-9ca6-a03d2db6ff17.png)
I can continue training before, I don’t know what I changed, I can’t continue training.

darknet.exe detector train data/obj.data yolov3-tiny_obj.cfg yolov3-tiny_obj_14000.weights -don't_show 
Still can not work
@AlexeyAB

**Comment by lyxlty** [comment]
![image](https://user-images.githubusercontent.com/11659778/43772806-fb4e3bee-9a75-11e8-8ac0-12c3e33507e9.png)

**Comment by AlexeyAB** [comment]
I don't see any error. Is the system just halted?

**Comment by lyxlty** [comment]
yes, I repeat compile darknet  ,  :darknet.exe detector train data/obj.data yolov3-tiny_obj.cfg yolov3-tiny_obj_14000.weights still not work ,

**Comment by lyxlty** [comment]
![image](https://user-images.githubusercontent.com/11659778/43820617-51e49554-9b19-11e8-91e7-cabcf52a0b05.png)

**Comment by lyxlty** [comment]
Fortunately, I backed up a copy darknet ,it work ,thanks!!! if i have 10 classes gesture recongnition,I think this detect small detection , yolov3-tiny.cfg need to  where is change , exception classes 10, filters=45, 
thanks!!!
